CONTRIBUTING.md

```diff
@@ -283,7 +289,7 @@ There are two ways to run TensorFlow unit tests.
     execution time and the sharding
     [could create an overhead on the test execution](https://github.com/bazelbuild/bazel/issues/2113#issuecomment-264054799).
 
-2.  Using [Docker](https://www.docker.com) and TensorFlow's CI scripts.
+3.  Using [Docker](https://www.docker.com) and TensorFlow's CI scripts.
```

Comment:
Markdown allows using the same number in all elements of the lists, and renumbers them on rendering to HTML, so this is not strictly needed